pkg/elemental/models/elemental-resource.js

```diff
@@ -1,8 +1,32 @@
 import SteveModel from '@shell/plugins/steve/steve-class';
+import { clone } from '@shell/utils/object';
 import { createElementalRoute } from '../utils/custom-routing';
+import { ELEMENTAL_PRODUCT_NAME } from '../config/elemental-types';
 
 export default class ElementalResource extends SteveModel {
   get listLocation() {
     return createElementalRoute('resource', { resource: this.type });
   }
+
+  get detailLocation() {
+    const detailLocation = clone(this._detailLocation);
+
+    detailLocation.params.resource = this.type;
+
+    if (detailLocation.name.includes('namespace')) {
```

Comment:
Only a comment.

Under the hood generic components check the schema to gate features on the type being namespaced

```
const schema = this.$getters['schemaFor'](this.type);
schema?.attributes?.namespaced
```

This is elemental world though, so i'm not sure how encompassing that is. So this is just something to be aware of if you need to do something similar in the future

Reply:
done